assets/urls.yaml

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We can save some verbosity from https://github.com/google/licensecheck/pull/52 if it gets merged